graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/IndicesAdapterES7.java

```diff
@@ -435,15 +440,24 @@ public IndicesBlockStatus getIndicesBlocksStatus(final List<String> indices) {
         if (indices == null || indices.isEmpty()) {
             throw new IllegalArgumentException("Expecting list of indices with at least one index present.");
         }
-        final GetSettingsRequest getSettingsRequest = new GetSettingsRequest()
-                .indices(indices.toArray(new String[]{}))
-                .indicesOptions(IndicesOptions.fromOptions(false, true, true, true))
-                .names(new String[]{});
 
-        return client.execute((c, requestOptions) -> {
-            final GetSettingsResponse settingsResponse = c.indices().getSettings(getSettingsRequest, requestOptions);
-            return BlockSettingsParser.parseBlockSettings(settingsResponse);
-        });
+        if(String.join(",", indices).length() > MAX_INDICES_URL_LENGTH) {
```

Comment:
I think we should safe some buffer here, as there will also be the path + additional query parameters (+ potential url encoding overhead?). Let's play it safe and reduce `MAX_INDICES_URL_LENGTH` to 2500?

Reply:
the max is 4096 - so I thought 3000 is already a buffer ;-)